spec/factories/application_choice.rb

```diff
@@ -59,6 +59,7 @@
     trait :withdrawn do
       status { :withdrawn }
       withdrawn_at { Time.zone.now }
+      withdrawn_or_declined_for_candidate_by_provider { true }
```

Comment:
@cwrw wasn't 100% on this one – an application could enter this state from either a candidate or provider action(?)

Reply:
I think we leave this false, and create a dedicated trait to withdrawn at candidate request and set to true? unless its not used anywhere then happy to keep this false